### PR TITLE
TS: do we need pageActivate ?

### PR DESCRIPTION
**Pull request by dron0gus**

### CODE REVIEW

**Comment by dron0gus** [comment]
@rusefillc is it some king of rudiment that can be safely removed?

**Comment by rusefillc** [comment]
@dron0gus yes, we do not really use pages. Actually I think you were requesting that we ADD pages https://github.com/rusefi/rusefi/issues/6046?

**Comment by rusefillc** [comment]
testing https://github.com/rusefi/rusefi/commit/fce0839ba4b47b49285b52744b38d21d43c2ec10

File: firmware/integration/ts_protocol.txt [hunk 1]
```diff
@@ -59,6 +56,7 @@
 #define TS_RESPONSE_FRAMING_ERROR 0x8D
 ! 0x52 82
 #define TS_READ_COMMAND 'R'
+#define TS_PAGE_READ_COMMAND 'r'
```
Comment (rusefillc) [inline firmware/integration/ts_protocol.txt:59]: Why is this line being added?

**Comment by dron0gus** [comment]
Yes, I want to add pages, but we need no activate page command. We have enough resources to keep all pages in RAM.

File: firmware/integration/ts_protocol.txt (hunk 1 above)
Comment (dron0gus) [reply firmware/integration/ts_protocol.txt]: Fixed

**Pull request merged**